spec/observables/from-promise-spec.js

```diff
@@ -1,6 +1,9 @@
 /* globals describe, it, expect */
 var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
+var isNode = typeof process === 'object' &&
+             process + '' === '[object process]' &&
+             typeof window === 'undefined';
```

Comment:
> but I suppose they could

: It is very correct, what I did above was simple check if process exists. Updating PR as suggested.